saleor/webhook/serializers.py

```diff
@@ -46,11 +46,16 @@ def serialize_product_or_variant_attributes(
 ) -> List[Dict]:
     data = []
 
-    def _prepare_reference(attribute, attr_slug):
+    def _prepare_reference(attribute, attr_value):
         if attribute.input_type != AttributeInputType.REFERENCE:
             return
+        if attribute.entity_type == AttributeEntityType.PAGE:
+            reference_pk = attr_value.reference_page_id
+        elif attribute.entity_type == AttributeEntityType.PRODUCT:
+            reference_pk = attr_value.reference_product_id
+        else:
+            reference_pk = attr_value.slug.split("_")[1]
```

Comment:
In which case would this logic be used? We shouldn't have to split the slugs anymore. I think we should rather return None.